### PR TITLE
[14.0][MIG] oca_custom: Migration to 14.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT30003

Depends on:
  - [x] github_connector: https://github.com/OCA/interface-github/pull/73

### CODE REVIEW

**Review by victoralmau — APPROVED** [review]

File: oca_custom/README.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Known issues / Roadmap
+======================
+
+* Make the github_login field from connector_github editable so it can be set
```
Comment (joao-p-marques) [inline oca_custom/README.rst]: This is already done by the module, right? Can this point be removed?

File: oca_custom/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    github_login = fields.Char(readonly=False)
```
Comment (joao-p-marques) [inline oca_custom/models/res_partner.py]: This field was changed in the refactor to v14. Now it is called `github_name`. IMHO, that makes more sense, as it is a field shared across all github related models, and is not always a “login”. I have done migration scripts in the refactor, so you would just need to change this to:
```suggestion
    github_name = fields.Char(readonly=False)
```

**Review by joao-p-marques — CHANGES_REQUESTED** [review]
Code review

**Comment by ernestotejeda** [comment]
@joao-p-marques changes done

**Review by CarlosRoca13 — APPROVED** [review]

**Review by joao-p-marques — APPROVED** [review]

**Pull request merged**